libbeat/idxmgmt/ilm/client_handler.go

```diff
@@ -205,6 +207,27 @@ func (h *esClientHandler) queryFeatures(to interface{}) (int, error) {
 	return status, nil
 }
 
-func (h *esClientHandler) access() ESClient {
-	return h.client
+func (h *stdoutClientHandler) ILMEnabled(mode Mode) (bool, error) {
```

Comment:
We should renamed `Enabled` to `Check` or `CheckEnabled`, or `CheckAvailable`. The name `Enabled` suggests that we should return a bool only.

Reply:
I can rename it.